### DISCUSSION TITLE
Welcome to AffiCodes-Hacktoberfest2024 Discussions!

### DISCUSSION THREAD

**Discussion by DevAffan** [DevAffan/AffiCodes#118] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by varshalohakre** [comment]
i have pull request the changes in BasicQuestions folder you have accepted it today but on my hacktoberfest profile it is showing non participating pr may i know the reason im contributing for first time.

**Comment by DevAffan** [comment]
Hi @varshalohakre , Thank you for letting me know. Please refer to the latest announcement topic.
If you are seeing PR is excluded than there is an issue from the repo which reported by Hachtoberfest jsut today.

Thank you.
